README.md

```diff
@@ -82,8 +90,10 @@ We use [SemVer](http://semver.org/) for versioning. See the [project releases](h
 
 Please read [CONTRIBUTING.md](CONTRIBUTING.md) for contribution and pull request protocol.
 
-Future implementations may include the number of yes/no votes for each validator and other data points. Proposed requirements and notes on this issue are described in RFC9 [Statistics of ballots](https://github.com/poanetwork/RFC/issues/9).
+Contributors should look into [issues](https://github.com/poanetwork/poa-ballot-stats/issues) or reference RFC9 [Statistics of ballots](https://github.com/poanetwork/RFC/issues/9) for additional information.
 
 ## License
 
-This project is licensed under the GNU Lesser General Public License [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0). See the [LICENSE](LICENSE) file for details.
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+
+This project is licensed under the GNU Lesser General Public License. See the [LICENSE](LICENSE) file for details.
```

Comment:
Maybe the text should also specify the version "v3.0", since the licenses are quite different.